### ISSUE TITLE
Cannot zoom on a ScrollView using the mouse/trackpad

### ISSUE THREAD

**Issue by jpnickolas** [microsoft/react-native-macos#705]
If you have the following code snippet, you can change the zoomScale by manually updating `zoom`, but the user cannot pinch zoom using their trackpad.

```
<ScrollView
maximumZoomScale={MAX_ZOOM}
minimumZoomScale={MIN_ZOOM}
zoomScale={zoom}>
</ScrollView>
```

This is something that works on windows, and is hopefully as simple as setting `allowsMagnification` on `NSScrollView`.

**Comment by chrisglein** [comment]
Seems like `allowsMagnification` is a macOS thing, not an iOS thing. Might just need to expose that prop. Does this work as expected on iOS? (well, for pinch, not for mouse obviously) Or rather, is it expected to be a macOS specific behavior?

**Comment by Saadnajmi** [comment]
Going to reopen this of this is indeed as simple as setting one native prop to true

**Comment by Saadnajmi** [comment]
I think we fixed this with 0.71? Could you give it a try?
